lib/runner.js

```diff
@@ -714,3 +718,19 @@ function extraGlobals() {
 
  return [];
 }
+
```

Comment:
@boneskull Their constructor is defined here
https://github.com/angular/angular.js/blob/master/src/ngMock/angular-mocks.js#L2371
and used here
https://github.com/angular/angular.js/blob/master/src/ngMock/angular-mocks.js#L2417
It serves a dual purpose for them: to extend the error stack of the original error, and to reuse `toString`.